Makefile

```
@@ -8,16 +8,13 @@ default: lint test
 lint:
 	golangci-lint run \
 		--disable-all \
-		--exclude-use-default=false \
+		--exclude-use-default=true \
```

Let's keep it aligned with other backend repo linter settings. This should do it:
```suggestion
		--exclude-use-default=false \
		--exclude=package-comments \
```

---

souds good